### PR TITLE
MV3 click to load support

### PR DESCRIPTION
**Pull request by ladamski**
https://app.asana.com/0/72649045549333/1202839073279061/f

### CODE REVIEW

**Comment by ladamski** [comment]
@jonathanKingston would appreciate a pass on this, should be complete but still working through some feedback on ddg2dnr and extension PRs so there's some chance this might change (a bit).

File: inject/chrome.js [hunk 1]
```diff
@@ -113,8 +113,8 @@ function init () {
     window.addEventListener('sendMessage', (m) => {
         const messageType = m.detail.messageType
         chrome.runtime.sendMessage(m && m.detail, response => {
-            const msg = { type: messageType, response }
-            const stringifiedArgs = JSON.stringify(msg)
+            const msg = { func: messageType, response }
+            const stringifiedArgs = JSON.stringify({ detail: msg })
```
[the remark this answers is not in the document]
Comment (jonathanKingston) [reply inject/chrome.js:117]: Also doesn't this break the cookie update messages also?

**Review by jonathanKingston — CHANGES_REQUESTED** [review]
Unless I'm misunderstanding the code I think there's cookie and Firefox issues here.

File: inject/chrome.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ladamski) [reply inject/chrome.js:117]: Thanks good catch!  I had test this earlier but then broke it with subsequent updates.  Now fixed again.  

I tested the content scope script cookie updates in MV2/MV3/FF (via console spam) and it still works; they don't go through this proxy AFAICT.

```
diff --git a/src/features/cookie.js b/src/features/cookie.js
index 73954cc..1848085 100644
--- a/src/features/cookie.js
+++ b/src/features/cookie.js
@@ -188,6 +188,7 @@ export function init (args) {
 }

 export function update (args) {
+    console.warn('cookie update', args && args.trackerDefinition && args.hostname)
     if (args.trackerDefinition) {
         trackerHosts.add(args.hostname)
     }
```

**Review by jonathanKingston — APPROVED** [review]

**Comment by jonathanKingston** [comment]
To merge you'll have to rebuild the inject file for chrome.

**Review by jonathanKingston — APPROVED** [review]

**Pull request merged**